packages/data/src/redux-store/index.js

```diff
@@ -155,6 +155,7 @@ export default function createReduxStore( key, options ) {
 			Object.assign( privateSelectors, selectors );
 		},
 	};
+	const privateListeners = new Set();
```

Comment:
guessing that this passed all the tests because it is unlikely that changes only occurred in one store, so the paths that are tested both produced situations where a combined array happened to work. plausibly there could be situations where it isn't fine.

or maybe the fact that these are all hooks and extra calls are/should be benign is fine.